tests/baselines/reference/exportBinding.errors.txt

```diff
@@ -0,0 +1,27 @@
+tests/cases/conformance/es6/modules/exportConsts.ts(3,16): error TS2448: Block-scoped variable 'x' used before its declaration.
+tests/cases/conformance/es6/modules/exportConsts.ts(3,16): error TS2454: Variable 'x' is used before being assigned.
+tests/cases/conformance/es6/modules/exportVars.ts(3,16): error TS2454: Variable 'y' is used before being assigned.
+
+
+==== tests/cases/conformance/es6/modules/exportConsts.ts (2 errors) ====
+    export { x }
+    export { x as xx }
+    export default x;

```

Comment:
According to what I'm seeing in the spec, I believe that is correct. The declaration evaluates and may have a side-effect, whereas the others are simple bindings.